python/cudf_polars/cudf_polars/dsl/expr.py

```diff
@@ -612,31 +635,13 @@ def do_evaluate(
                     (c.obj for c in columns),
                 )
             )
-        elif self.name == pl_expr.BooleanFunction.IsBetween:
```

Comment:
This now never appears in our IR because during translation time we produce binops.